# PGx-MR-from-EHR-GWAS
Full results from Smith et al. 2024: Identifying DNA methylation sites affecting drug response using electronic health recordâ€“derived GWAS summary statistics


Adverse drug responses (ADRs) result in over 7,000 deaths annually. Pharmacogenomic studies have shown that many ADRs are partially attributable to genetics. However, emerging data suggest that epigenetic mechanisms, such as DNA methylation (DNAm) also contribute to this variance. Understanding the impact of DNA methylation on drug response may minimize ADRs and improve the personalization of drug regimens. In this work, we identify DNA methylation sites that likely impact drug response phenotypes for anticoagulant and cardiometabolic drugs. We use instrumental variable analysis to integrate genome-wide association study (GWAS) summary statistics derived from electronic health records (EHRs) within the U.K. Biobank (UKBB) with methylation quantitative trait loci (mQTL) data from the Genetics of DNA Methylation Consortium (GoDMC). This approach allows us to achieve a robust sample size for our analysis largest publicly available pharmacogenomic GWAS. For warfarin, we find 71 DNAm sites. Of those, 8 are near the gene VKORC1 and 48 are on chromosome 6 near the human leukocyte antigen (HLA) gene family. We also find 2 DNA methylation sites near the genes CYP2C9 and CYP2C19. For statins, we identify 17 DNAm sites. Eight are near the low-density lipoprotein cholesterol carrier protein APOB. We find no significant results for metformin.


Included in this repository are the extended results in supplemental files for statins and warfarin. Email any questions to smithdel@stanford.edu.
